tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pytest
+
+root_dir = os.getcwd()
+module_name = "iotedgeFilterModule"
+file_list = ["requirements.txt", "module.json", "main.py", "Dockerfile.arm32v7",
+             "Dockerfile.amd64.debug", "Dockerfile.amd64", ".gitignore"]
+
+
+@pytest.fixture(scope="module")
+def test_install_iotedgemodule(request):
+    installmodulecommand = "cookiecutter --no-input https://github.com/Azure/cookiecutter-azure-iot-edge-module module_name=" + \
+        module_name + " image_repository=localhost:5000/"+module_name+" --checkout master"
+    pipe = os.popen(installmodulecommand)
+    yield pipe.readlines()
+    pipe.close()
+
+
+@pytest.fixture(scope="module", params=file_list)
+def test_check_file_exist(request):
```

Comment:
test_check_file_exist should not be a fixture. test_check_file_exist and test_check_iotedgemodule_exist should be merged to one test case

Reply:
updated